### ISSUE TITLE
AlphaFold 2 learning rate question

### ISSUE THREAD

**Issue by gahdritz** [deepmind/alphafold#390]
In the supplement, it's claimed that 

```
We linearly increase (warm-up) the learning rate over the first 0.128 * 10^6 samples, and further decrease the learning rate by a factor of 0.95 after 6.4 * 10^6 samples.
```

Two questions:

1. The decrease mentioned above is not recorded in Table 4, where the learning rate is decreased only after 10M samples, at the beginning of the finetuning stage. I wanted to confirm that the decrease after 6.4  * 10^6 samples actually happened.
2. The wording describing the decrease is a little ambiguous: is the new learning rate after the decrease (0.001 * 0.95) or (0.001 * 0.05)?

**Comment by guolinke** [comment]
I think I asked the same question. https://github.com/deepmind/alphafold/issues/380

**Comment by gahdritz** [comment]
Yes, you're right. Thanks!
